### ISSUE TITLE
Preferred countries flicker on load

### ISSUE THREAD

**Issue by lihi-intsights** [jackocnr/intl-tel-input#772]
When loading the page with input that already has phone number connected to it, we see the preferred country and only after it the right flag.

### Steps to reproduce
1.  Create input that has a model of phone number.
2.  Refresh the page.

### Expected behaviour
On page load, the flag should be the one of the model.

### Actual behaviour
On page load, there is a flicker of the preferred country, and then appears the phone number flag.

**Comment by lihi-intsights** [comment]
First, thanks a lot for the quick reply.

I put my code here: https://codepen.io/Lihisham/pen/rrrbBm
It is important to say that the flicker happens really fast in the test
code because it is not so heavy, but in my project you can really notice
that.
I'm attaching video that captures the US flag before the Mexican (0:05).

Again, thank you!

**Comment by lihi-intsights** [comment]
So as you said, I set the input value before the initialization of the plugin, but now there is a flicker of the number with the prefix.
You can watch it here: https://codepen.io/Lihisham/pen/rrrbBm

**Comment by jackocnr** [comment]
Yeah that's a good point, but in my opinion it's an acceptable compromise. I think it would be even better if you could inject the input value in the markup, because currently technically you're showing an empty input until the JS runs, so on a slow connection users might see that state too.

Whatever you do you're going to start with a regular input, and then the flag is going to be injected when the JS runs, so there's always going to be a visual jump. IMO it's acceptable to have the number jump slightly at this point as it gets reformatted. Better to group these things together.

AFAICS the only way to do this "perfectly", with no jumps, would be to completely hide the input element initially (maybe show a loading spinner or something) until you have initialised the plugin and then show it.
